.gitignore

```diff
@@ -1,9 +1,68 @@
-/.idea/
-/project/boot/
-/project/plugins/project
+*.class
+*.log
+
+# sbt specific
+.sbtopts
+.cache/
+.history/
+.lib/
+dist/*
 target/
 lib_managed/
 src_managed/
-test-output/
+project/boot/
+project/plugins/project/
+
+# Scala-IDE specific
+.scala_dependencies
+.worksheet
+
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm
+
 *.iml
-.classpath_nb
+
+## Directory-based project format:
+.idea/
+# if you remove the above rule, at least ignore the following:
```

Comment:
This is a standard gitignore configuration for Intellij. It's generally best to leave intellij config files out of a repo, but some folks (https://github.com/samtools/htsjdk) like to have them in.